tests/python_package_test/test_dask.py

```diff
@@ -42,7 +43,8 @@
 
 pytestmark = [
     pytest.mark.skipif(getenv('TASK', '') == 'mpi', reason='Fails to run with MPI interface'),
-    pytest.mark.skipif(getenv('TASK', '') == 'gpu', reason='Fails to run with GPU interface')
+    pytest.mark.skipif(getenv('TASK', '') == 'gpu', reason='Fails to run with GPU interface'),
+    pytest.mark.skipif(machine() != 'x86_64', reason='Fails to run with non-x86_64 architecture')
```

Comment:
In what way did the Dask tests fail?

Reply:
I don't remember, sorry! I marked Dask tests to be skipped at the very begging of enabling QEMU and cannot find logs for runs where they were enabled. I think we can investigate Dask support for aarch64 in a follow-up PR. WDYT?

Reply:
yep it's ok, can be a follow-up